ledger/participant-integration-api/src/main/scala/platform/store/DbType.scala

```diff
@@ -50,17 +50,4 @@ object DbType {
     case _ =>
       sys.error(s"JDBC URL doesn't match any supported databases (h2, pg, oracle)")
   }
-
-  sealed trait AsyncCommitMode {
-    def setting: String
-  }
-  object SynchronousCommit extends AsyncCommitMode {
-    override val setting: String = "ON"
-  }
-  object AsynchronousCommit extends AsyncCommitMode {
-    override val setting: String = "OFF"
-  }
-  object LocalSynchronousCommit extends AsyncCommitMode {
```

Comment:
AFAICR this was introduced by Canton. Please make sure it is not used before removing.

Reply:
It is a "almost" isomorphic type to `SynchronousCommitValue` so I removed this additional layer of indirection. @mziolekda also challenged if we want to keep using `SynchronousCommitValue` actually

Reply:
Let me double check - I could have confused something.

Reply:
`supportsAsynchronousCommits` is not used in main as well. I also checked canton and oem kit.